components/brave_new_tab_ui/widgets/ftx/components/preOptIn.tsx

```diff
@@ -32,13 +41,13 @@ export default function PreOptIn (props: Props) {
       <BasicBox isFlex={true} $gap={10} justify='flex-end'>
         <OptionButton
           isSelected={!isDotUsSelected}
-          onClick={setIsDotUsSelected.bind(undefined, false)}
+          onClick={setUserSetIsUSRegion.bind(undefined, false)}
```

Comment:
Not necessarily a bug - as the correct default is picked. But (with US region) if I click this one for example (.com), it doesn't save. When I click, it does correctly display below `FTX.com`... and after quit / restarting, it's resetting to `FTX.us`.  Is this OK? Or should it be setting `props.ftx.ftxHost`?

Reply:
Yes I think it's unnecessary to persist before the user logs in. This is not something that toggles anything until the user connects.